Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.cpp

```diff
@@ -1958,6 +1969,30 @@ namespace AzToolsFramework
         }
     }
 
+    void EntityPropertyEditor::InvalidatePropertyDisplayForComponent(AZ::EntityComponentIdPair entityComponentIdPair, PropertyModificationRefreshLevel level)
+    {
+        if (level == Refresh_None)
+        {
+            return;
+        }
+
+        if (m_selectedEntityComponentIds.find(entityComponentIdPair) == m_selectedEntityComponentIds.end())
+        {
+            return;
+        }
+
+        if (!m_isAlreadyQueuedRefresh)
```

Comment:
nit: 
```suggestion
        if ((level != Refresh_None) && 
             (m_selectedEntityComponentIds.find(entityComponentIdPair) != m_selectedEntityComponentIds.end()) &&
            (!m_isAlreadyQueuedRefresh))
```

Reply:
I actually think the early return code is a lot easier to read and less dense.  Thoughts?

Reply:
I might even change it so that its 
```cpp
if (m_isAlreadyQueuedRefresh)
{
    return;
}
```
just to follow the pattern...

Reply:
I personally like early returns as well for readability, but the comment was just to make the read of the function consistent. I like following the pattern tbh